src/test/ui/const-generics/issues/issue-70125-2.rs

```diff
@@ -13,4 +13,4 @@ trait Foo<const X: usize> {
     }
 }
 
-impl Foo<{3}> for () {}
+impl Foo<3> for () {}
```

Comment:
Let's avoid changing this regression test in case the braces were relevant.

Reply:
They weren't: I think it's fine to remove all the braces in the existing const generics tests.

Reply:
Ah; well if you're sure then OK. :)